src/core/templates.ts

```diff
@@ -220,4 +222,29 @@ export class Templates {
         return { 'envoy-proxy-secret': nodeAlias }
     }
   }
+
+  /**
+   * Parsed and validates the unparsed value of flag clusterMappings
+   *
+   * @param unparsed - value of flag clusterMappings
+   */
+  public static parseContextCluster (unparsed: string): ContextClusterStructure {
+    const mapping = {}
+
+    unparsed.split(',').forEach((data) => {
+      const [context, cluster] = data.split('=') as [Context, Cluster]
+
+      if (!context || typeof context !== 'string') {
+        throw new SoloError('Invalid context in context-cluster', null, { data })
+      }
+
+      if (!cluster || typeof cluster !== 'string') {
+        throw new SoloError('Invalid cluster in context-cluster', null, { data })
```

Comment:
maybe a better error message, and explain the format so that they don't have to do a `--help`.

```
❯ solo deployment create --namespace solo-e2e --deployment-name solo-e2e --deployment-clusters kind-solo-e2e --context-cluster kind-solo-e2e

******************************* Solo *********************************************
Version			: 0.99.0
Kubernetes Context	: kind-solo-e2e
Kubernetes Cluster	: kind-solo-e2e
Kubernetes Namespace	: solo-e2e
**********************************************************************************
(node:89410) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
✖ Invalid cluster in context-cluster
◼ Prompt local configuration
◼ Validate cluster connections
◼ Create remote config
*********************************** ERROR *****************************************
Error installing chart solo-deployment
***********************************************************************************
```